bluemira/equilibria/coils/_grouping.py

```diff
@@ -283,6 +284,26 @@ def from_group_vecs(cls, eqdsk: EQDSKInterface):
         pfcoils = []
         cscoils = []
         passivecoils = []
+        if eqdsk.ncoil < 1:
+            grid = Grid.from_eqdsk(eqdsk)
+            dum_xc = [np.min(grid.x), np.max(grid.x), np.max(grid.x), np.min(grid.x)]
+            dum_zc = [np.min(grid.z), np.min(grid.z), np.max(grid.z), np.max(grid.z)]
+            for i in range(4):
+                coil = Coil(
+                    dum_xc[i],
+                    dum_zc[i],
+                    current=0,
+                    dx=0,
+                    dz=0,
+                    ctype="PF",
```

Comment:
Perhaps we could have a dummy type? And make sure they don't get plotted?

Reply:
New ctype=DUM added and coils do not plot.